Daramola98/WEconnectApp
# ##3894881 Build User interfaces for WEconnect Application

#### What does this PR do?
Have working user interface templates for WEconnect application

#### Description of Task to be completed?
The following pages should be working
index.html
signUp.html
login.html
updateBusiness.html
registerBusiness.html
businessProfile.html
businessesListing.html

#### How should this be manually tested?
Visit the webpages at https://daramola98.github.io/WEconnectApp/template and navigate through all the pages
To view pages that require authentication sign in with any username and password
Check for responsiveness and cross-browser compatibility

#### What are the relevant pivotal tracker stories?
#155544106, #155543535, #155544277, #155544795, #155544883, #155545028, #155549629
